### PR TITLE
Expose cv_policy_explicit in Python and Python docs.

### PR DESCRIPTION
**Pull request by brenthuisman**

### CODE REVIEW

File: doc/python/morphology.rst [hunk 1]
```diff
@@ -522,6 +537,9 @@ constitute part of the CV boundary point set.
 
 .. _pyneuroml:
 
+NeuroML
```
Comment (noraabiakar) [inline doc/python/morphology.rst:540]: There's also an SWC section here somewhere. If this doesn't have a header, that probably doesn't either.

Comment (brenthuisman) [reply doc/python/morphology.rst:543]: It was all a bit messed up, I fixed the ordering and added headings to the file formats.

Comment (bcumming) [reply doc/python/morphology.rst:543]: I checked over the file, and the new ordering (input file formats at the bottom of the file) is more logical.

**Review by bcumming — APPROVED** [review]
looks good!

**Pull request merged**